templates/style/rustdoc.scss

```
@@ -1,13 +1,29 @@
 // FIXME: Use modules
 @import "vars", "navbar", "themes", "fa";
 
+body.rustdoc-age {
```

```suggestion
body.rustdoc-page {
```

I assume by the next rule?

---

Woups, indeed!